src/task.js

```diff
@@ -37,8 +53,14 @@ class Task {
 
       const taskComplete = document.createElement('input');
       taskComplete.type = 'checkbox';
+      taskComplete.className = 'is-completed';
       taskComplete.checked = element.completed;
 
+      taskComplete.addEventListener('change', () => {
+        element.completed = true;
+        this.saveData();
+      });
```

Comment:
- [ ] Good job working on this, but toggling the checkbox does not work as expected. When you click the checkbox it makes a task completed but when you need to toggle back it does not work. This might be because on line 60 you are explicitly setting the completed to true.